<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>399</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Report2csv 0.0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>csv.png</normaloff>csv.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>导入</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <column>
          <property name="text">
           <string>零件号</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>零件名</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ICMD/ICM</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ICMC/IQV</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(13, 110, 253);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>导入报告</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>导出</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(13, 110, 253);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>选择路径</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>progressBar</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
